tls/s2n_ktls.h

```diff
@@ -31,4 +29,4 @@ typedef enum {
     S2N_KTLS_MODE_DUPLEX,
 } s2n_ktls_mode;
 
-int s2n_config_set_ktls_mode(struct s2n_config *config, s2n_ktls_mode ktls_mode);
+int s2n_ktls_enable(struct s2n_connection *conn, s2n_ktls_mode ktls_mode);
```

Comment:
I know you're naming this after the module, but for public APIs we generally name them after the struct they operate on. So this should be s2n_connection_enable_ktls.